content/post/elm-at-rakuten/index.md

```diff
@@ -1,91 +1,99 @@
 ---
-title: "Elm in Rakuten"
-date: 2021-01-12T00:00:00+09:00
-hero: ./images/elm-in-rakuten.png
+title: "Elm at Rakuten"
+date: 2021-07-13T00:00:00+09:00
+hero: /post/elm-at-rakuten/images/elm-at-rakuten.png
 excerpt: "Our story of adopting the Elm language in Rakuten. The lessons we learned, and our likes and dislikes."
 timeToRead: 5
 authors:
   - Luca Mugnaini
 draft: false
+seo:
+  canonical: https://dev.to/lucamug/elm-6m8
```

Comment:
...once we should change this before we publish...

Reply:
Should I change now?

Reply:
yeah let's do it!

Reply:
Done